datasets/brk/dataset.json

```diff
@@ -2,106 +2,73 @@
   "type": "dataset",
   "id": "brk",
   "title": "brk",
-  "status": "niet_beschikbaar",
-  "version": "0.0.1",
+  "status": "beschikbaar",
+  "version": "1.0.0",
   "crs": "EPSG:28992",
   "identifier": "identificatie",
-  "owner": "Gemeente Amsterdam",
-  "creator": "bronhouder onbekend",
+  "owner": "Gemeente Amsterdam, afdeling Basisinformatie",
+  "creator": "Kadaster",
   "publisher": "Datateam Basis- en Kernregistraties",
-  "auth": "OPENBAAR",
-  "authorizationGrantor": "OIS",
+  "auth": "BRK/RO",
```

Comment:
Even voor de zekerheid. Je sluit hiermee een aantal tabellen af, die momenteel vrij toegankelijk zijn. Zoals https://api.data.amsterdam.nl/v1/brk/gemeentes/ Is dit de bedoeling? Het is mogelijk dat afnemers hier last van krijgen.

Reply:
Dank, RO blijkt alleen op kad objecten te slaan